airbyte-cdk/python/airbyte_cdk/sources/concurrent_source/concurrent_source_adapter.py

```diff
@@ -53,19 +53,7 @@ def _select_abstract_streams(self, config: Mapping[str, Any], configured_catalog
         for configured_stream in configured_catalog.streams:
             stream_instance = stream_name_to_instance.get(configured_stream.stream.name)
             if not stream_instance:
-                if not self.raise_exception_on_missing_stream:
```

Comment:
are we deleting this logic because it redundant to the code in `abstract_source.py`?

Reply:
yes, all non-existent streams will be filtered in line 38 above and then handled with the logic in `abstract_source.py`

Reply:
yes, all non-existent streams will be filtered in line 38 above and then handled with the logic in `abstract_source.py`